webapp/src/components/ProgramCalendar.tsx

```diff
@@ -27,6 +29,7 @@ const activitiesForCalendar = (
         end: new Date(activity.end_time),
         description: decodeHTML(activity.description_text),
         allDay: !activity.duration,
+        theProgram: correspondingProgramTitle//activity.program_id.toString()
```

Comment:
Not only should comments like these be removed before a pull request is created, but I don't believe the comment is correct, since you want to save the program's name, not its ID.

Also, I would modify this member's name from `theProgram` to something like `programTitle`, or I would name it `program` and just attach the Program. But I think `programTitle` is the better approach to naming this member.